src/Symfony/Component/HttpKernel/Profiler/BaseMemcacheProfilerStorage.php

```diff
@@ -33,6 +36,8 @@
      */
     public function __construct($dsn, $username = '', $password = '', $lifetime = 86400)
     {
+        @trigger_error('The '.__CLASS__.' class is deprecated since Symfony 2.8 and will be removed in 3.0. Use FileProfilerStorage instead.', E_USER_DEPRECATED);
```

Comment:
You will have to change __CLASS__ by __NAMESPACE__ + the name of the class. I can one downside: when using a sub-class as the class name won't be the right one.